README.md

```diff
@@ -98,6 +98,10 @@ protocols:
         exitcode?: number
           # Mocked exitcode of the command.
           # Default: 0
+        filetree: string
+          # List of files and folders that your script expects to exist.
```

Comment:
Currently the tool will still find files from the filesystem, right? So maybe this should mention something about mocked files. Maybe `List of files and folders that are going to be mocked to exist.`